neuro_flow/cli.py

```diff
@@ -308,3 +308,7 @@ async def show(
             BatchRunner(config_dir, client, storage)
         )
         await runner.logs(bake_id, task_id, attempt_no=attempt, raw=raw)
+
+
+if __name__ == "__main__":
```

Comment:
This allows to run our tool using `python -m neuro_flow.cli args_here`. This allows me to debug code in Pycharm. Should I revert this changes? Maybe I'm missing some more elegant way to debug the code.

Reply:
I'm fine with it if it makes your life easier.
I don't use pycharm but debug directly with pdb++